rust_src/.dir-locals.el

```diff
@@ -0,0 +1,3 @@
+((rust-mode
+  (racer-rust-src-path . "/home/shaleh/\.rustup/toolchains/nightly-2018-02-03-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/src")))
+
```

Comment:
I don't think you meant to check this in :)

Reply:
Indeed. I thought this was covered with gitignore. Bah!